src/main/java/pl/company/foodatu/plans/PlansController.java

```diff
@@ -0,0 +1,30 @@
+package pl.company.foodatu.plans;
+
+import org.springframework.web.bind.annotation.*;
+import pl.company.foodatu.plans.domain.PlansFacade;
+import pl.company.foodatu.plans.dto.UserId;
+import pl.company.foodatu.plans.dto.Meal;
+import pl.company.foodatu.plans.dto.PlanResponse;
+
+import java.time.LocalDate;
+
+@RequestMapping("/api/v1/")
+@RestController
+public class PlansController {
+
+    private PlansFacade plansFacade;
+
+    public PlansController(PlansFacade plansFacade) {
+        this.plansFacade = plansFacade;
+    }
+
+    @PostMapping("/{userId}/plans/{day}")
```

Comment:
```suggestion
    @PostMapping("/{userId}/plans/{day}/meals")
```

Please look through [Allegro REST guidelines](https://github.com/allegro/restapi-guideline). This is great place to get familiar with good practices